extra/2-piping.js

```diff
@@ -16,26 +16,28 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  return num1 + num2;
 }
 
-function multiply() {
-
+function multiply(num1, num2) {
+  return num1 * num2;
 }
 
-function format() {
-
+function format(num) {
+  return "\u00A3" + Number(num);
```

Comment:
you don't need the `Number` here as it turns `num` into a number (if it isn't already), but then by joining it to the string the whole value becomes a string. So this can be simplified to
```
return "\u00A3" + num;
```